localization/zh/chain/App.java

```diff
@@ -0,0 +1,93 @@
+@Getter
+public class Request {
+
+    private final RequestType requestType;
+    private final String requestDescription;
+    private boolean handled;
+
+    public Request(final RequestType requestType, final String requestDescription) {
+        this.requestType = Objects.requireNonNull(requestType);
```

Comment:
Instead of Objects.requireNonNull(), can we use @NonNull annotation? Please check once the implementation.